website/src/components/IconButton/IconButton.test.jsx

```diff
@@ -1,24 +1,26 @@
 import React from 'react';
 import renderer from 'react-test-renderer';
-import { shallow, mount, render, configure} from 'enzyme';
+import {
+  shallow,
```

Comment:
Maybe write more tests to try out different types of icons and see if they render properly